livekit-plugins/livekit-plugins-hume/livekit/plugins/hume/tts.py

```diff
@@ -80,29 +92,25 @@ def __init__(
         self,
         *,
         api_key: str | None = None,
-        utterance_options: NotGivenOr[PostedUtterance] = NOT_GIVEN,
-        context: PostedContext | None = None,
+        utterance_options: NotGivenOr[UtteranceOptions] = NOT_GIVEN,
+        context: ContextGenerationId | ContextUtterances | None = None,
         instant_mode: bool = True,
         audio_format: AudioFormat = AudioFormat.mp3,
         base_url: str = DEFAULT_BASE_URL,
         http_session: aiohttp.ClientSession | None = None,
     ):
         super().__init__(
-            capabilities=tts.TTSCapabilities(streaming=True),
+            capabilities=tts.TTSCapabilities(streaming=False),
```

Comment:
an alternate version of this PR would just be this single line. maybe better because it wouldn't be an interface change for developers but i think the way the interface is done now is a lot cleaner